app/helpers/companies_helper.rb

```diff
@@ -1,8 +1,8 @@
 module CompaniesHelper
 
   def company_complete? company
-    company&.name.blank? ||
-    (company&.region.blank? && company&.old_region.blank?) ? false : true
+    return false if ! company
```

Comment:
You're right about both methods.

The only reason I might think to retain these is that I imagine in the future we'll be asked to create a view showing _all_ companies, regardless of "complete" status.  For instance, to allow for deleting abandoned efforts to create a company (which are cluttering up the DB with useless records).

However, that is not an argument to retain these methods.  We can cross that bridge when we come to it.  (should have given a cliché trigger warning ...)

I will remove the two methods.